### DISCUSSION TITLE
Linux Test Program

### DISCUSSION THREAD

**Discussion by mtgstuber** [funbiscuit/embedded-cli#40] [General]
I've ported the Win32 Example test program to run under a Linux console.  I'm planning on using the library in an embedded Linux environment, and I needed to prove to myself that I could make it work.  

That said, make files and cmake really aren't my thing, so I'm just compiling it by hand.  Is this interesting enough to submit?  If so, I'll make it pretty.  If not, I won't bother.

**Comment by funbiscuit** [comment]
Yes, please submit a PR, I'll add necessary cmake file to compile it along the library.

**Comment by mtgstuber** [comment]
Submitted.  Please advise if you have questions or concerns.
